app/helpers/number_helper.rb

```diff
@@ -0,0 +1,82 @@
+module NumberHelper
+  def number_to_human_size(number, *args)
+    options = args.extract_options!.reverse_merge(:significant => false, :precision => 1)
+    NumberHelper.handling_negatives(number) do |number|
+      super(number, options)
```

Comment:
This module gets included into ApplicationHelper; the same-named Rails built-in helper lives further up the inheritance chain, so that's what gets invoked when we `super`.